src/sql/base/browser/ui/table/table.ts

```diff
@@ -285,6 +286,7 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 
 		if (styles.tableHeaderBackground) {
 			content.push(`.monaco-table .${this.idPrefix} .slick-header .slick-header-column { background-color: ${styles.tableHeaderBackground}; }`);
+			content.push(`.monaco-table.${this.idPrefix} .slick-row .dynamic-cell-detail { background-color: ${styles.tableHeaderBackground} }`);
```

Comment:
Is this selector correct? You're checking if the idPrefix class is on the element with monaco-table class (see the one above)

Reply:
RowDetails view plugin draws a panel inside a monaco table rows. The full path to the panel is (with actual values):
.monaco-table.${this.idPrefix} > .${this.idPrefix}.ui-widget > slick-viewport > grid-canvas > ui-widget-content.slick-row.even > .dynamic-cell-detail
So, I don't see any issues here.

Reply:
I see, didn't know the id was on both.

Is there any issue with separating out the selectors though? I'd rather be consistent since otherwise someone might see this and wonder the same thing as I did. It's definitely not obvious that this would be the case.

Reply:
Ok. It makes sense.